src/sqs/queues.ts

```diff
@@ -9,12 +9,20 @@ const sqsQueues = {
 	backfill: new SqsQueue<BackfillMessagePayload>({ queueName: "backfill",
 		queueUrl: envVars.SQS_BACKFILL_QUEUE_URL,
 		queueRegion: envVars.SQS_BACKFILL_QUEUE_REGION,
-		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC}, backfillQueueMessageHandler),
+		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC,
+		timeoutSec: 10*60,
```

Comment:
10 minutes to process one message? This looks quite oof tbh... With our VisibilityTimeout set to 2 minutes (if I'm not mistaken), that's an open gate to duplicates... Am I missing something?

Reply:
Currently it is 10 minutes for the same queue on Redis.  We can adjust the timeout when we'll be connecting Installations queue logic to SQS

Reply:
I think we need to have same or bigger VisibilityTimeout in service descriptor then...

Reply:
Currently we change the visibility timeout as soon as we read the message so whatever we put in the Micros Descriptor will be overridden.

Anyway would be good to add "just in case". We can address it in the next PR.

Reply:
If the value in service-descriptor overwritten, I think we should call this out explicitly in it (add a comment maybe?)...
- [ ] add comment to service descriptor explaining that we are not using VisibilityTimeout value